whois-query/src/main/java/net/ripe/db/whois/query/rpki/RoutinatorDataProvider.java

```diff
@@ -0,0 +1,46 @@
+package net.ripe.db.whois.query.rpki;
```

Comment:
Don't put these classes in `whois-query` that package is for querying *whois*, instead put them in `whois-commons`

Reply:
moved to whois-commons